src/components/MyAccount/Settings/AccountSettingsUtils.ts

```diff
@@ -29,10 +33,12 @@ export const accountSettings = [
     term: "Notification preference",
   },
   {
-    field: "homeLibraryCode",
+    field: "homeLibrary",
     icon: "actionHome",
     term: "Home library",
-    description: (code) => getLibraryByCode(code)?.name,
+    description: (location) => {
```

Comment:
Small stylistic preference but the parentheses and return aren't necessary here.